src/Login.js

```diff
@@ -32,21 +50,28 @@ function Login() {
 }
 
 function login(email, password) {
+    var result;
     $.ajax({
         url: 'http://localhost/classask/src/php/login.php',
         type: 'GET',
+        async: false, // is this safe?
```

Comment:
Dont think you need this, you can probably get rid of it

Reply:
It was one of the ways I found that I could return values from the ajax file though

Reply:
so if the async isn't there, you cant return 1 or 0?

Reply:
ah okay, in that case, does that async flag have to be explicitly set? are ajax functions async by default?

Reply:
I think they are async by default yeah so it probably needs to be explicitly set to be false

Reply:
alright in that case, that looks good! should be safe as well :)